[openvino] build failure

### Operating system

Windows 8.1 x64

### Compiler

MSVC 19.16.27049.0

### Steps to reproduce the behavior

```Shell
.\vcpkg install opencv[contrib]:x64-windows
```


### Failure logs

```
-- Using cached openvinotoolkit-openvino-2023.1.0.tar.gz.
-- Cleaning sources at C:/CYH/vcpkg/buildtrees/openvino/src/2023.1.0-d4639b6ba4.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/CYH/vcpkg/downloads/openvinotoolkit-openvino-2023.1.0.tar.gz
-- Applying patch 001-disable-tools.patch
-- Applying patch 003-fix-find-onnx.patch
-- Applying patch 004-compilation-with-cpp17.patch
-- Applying patch 005-tflite-search.patch
-- Applying patch 007-macos-14.patch
-- Applying patch 008-dynamic-protubuf.patch
-- Applying patch 009-tensorflow-proto-odr.patch
-- Using source at C:/CYH/vcpkg/buildtrees/openvino/src/2023.1.0-d4639b6ba4.clean
-- Using cached oneapi-src-oneDNN-ec0b2ee85fc2a2dbdeec10035c5ef5813d8fb5ea.tar.gz.
-- Cleaning sources at C:/CYH/vcpkg/buildtrees/openvino/src/813d8fb5ea-f10e7a77bc.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/CYH/vcpkg/downloads/oneapi-src-oneDNN-ec0b2ee85fc2a2dbdeec10035c5ef5813d8fb5ea.tar.gz
-- Applying patch 006-onednn-gpu-build.patch
-- Using source at C:/CYH/vcpkg/buildtrees/openvino/src/813d8fb5ea-f10e7a77bc.clean
-- Using cached openvinotoolkit-oneDNN-a1aa20ca8f19465dc2fd18389953ed83798b2fd3.tar.gz.
-- Cleaning sources at C:/CYH/vcpkg/buildtrees/openvino/src/83798b2fd3-bd5f7c44d8.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/CYH/vcpkg/downloads/openvinotoolkit-oneDNN-a1aa20ca8f19465dc2fd18389953ed83798b2fd3.tar.gz
-- Using source at C:/CYH/vcpkg/buildtrees/openvino/src/83798b2fd3-bd5f7c44d8.clean
-- Using cached openvinotoolkit-mlas-c7c8a631315000f17c650af34431009d2f22129c.tar.gz.
-- Cleaning sources at C:/CYH/vcpkg/buildtrees/openvino/src/9d2f22129c-d6edabccf5.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/CYH/vcpkg/downloads/openvinotoolkit-mlas-c7c8a631315000f17c650af34431009d2f22129c.tar.gz
-- Using source at C:/CYH/vcpkg/buildtrees/openvino/src/9d2f22129c-d6edabccf5.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/CYH/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j5
    Working Directory: C:/CYH/vcpkg/buildtrees/openvino/x64-windows-dbg
    See logs for more information:
      C:\CYH\vcpkg\buildtrees\openvino\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/openvino/portfile.cmake:134 (vcpkg_cmake_install)
  scripts/ports.cmake:168 (include)



```
<details><summary>C:\CYH\vcpkg\buildtrees\openvino\install-x64-windows-dbg-out.log</summary>

```
[0/2] C:\CYH\vcpkg\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -P C:\CYH\vcpkg\buildtrees\openvino\x64-windows-dbg\CMakeFiles\VerifyGlobs.cmake

[1/3114] C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe   /TP -DIE_BUILD_POSTFIX=\"d\" -DIN_OV_COMPONENT -DOV_NATIVE_PARENT_PROJECT_ROOT_DIR=\"2023.1.0-d4639b6ba4.clean\" -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\util\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -std:c++14 -MDd /d1trimfile:C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\ /d1trimfile:C:/CYH/vcpkg/buildtrees/openvino/src/2023.1.0-d4639b6ba4.clean/ /showIncludes /Fosrc\common\util\CMakeFiles\openvino_util.dir\src\common_util.cpp.obj /FdC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\bin\intel64\Debug\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\util\src\common_util.cpp

[2/3114] C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe   /TP -DIE_BUILD_POSTFIX=\"d\" -DIN_OV_COMPONENT -DOV_NATIVE_PARENT_PROJECT_ROOT_DIR=\"2023.1.0-d4639b6ba4.clean\" -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\util\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -std:c++14 -MDd /d1trimfile:C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\ /d1trimfile:C:/CYH/vcpkg/buildtrees/openvino/src/2023.1.0-d4639b6ba4.clean/ /showIncludes /Fosrc\common\util\CMakeFiles\openvino_util.dir\src\env_util.cpp.obj /FdC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\bin\intel64\Debug\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\util\src\env_util.cpp

[3/3114] C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe   /TP -DIE_BUILD_POSTFIX=\"d\" -DIN_OV_COMPONENT -DOV_NATIVE_PARENT_PROJECT_ROOT_DIR=\"2023.1.0-d4639b6ba4.clean\" -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\itt\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\util\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -std:c++14 -MDd /d1trimfile:C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\ /d1trimfile:C:/CYH/vcpkg/buildtrees/openvino/src/2023.1.0-d4639b6ba4.clean/ /showIncludes /Fosrc\common\itt\CMakeFiles\openvino_itt.dir\src\itt.cpp.obj /FdC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\bin\intel64\Debug\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\itt\src\itt.cpp

...
Skipped 131558 lines
...
[16/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\message_patterns.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[17/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\ir_builder.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[18/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\conv\ir_builder.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[19/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\message.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[20/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\post_ops.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[21/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\slm_reduce_builder.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[22/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\core.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
[23/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\reduce.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[24/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\gemm\xe_hp_systolic_gemm.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[25/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\send_plan.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[26/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\gemm\loop_sequencer.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
[27/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\gemm\strategy_parser.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
[28/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\fma.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[29/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\gen9_simple_sum.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[30/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\jit_post_op_injector.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
[31/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\ir.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[32/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\epilogue.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[33/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\gemm_schedule.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[34/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ngen\ngen_register_decl.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
[35/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ngen\ngen_register_allocator.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
[36/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\jit_eltwise_injector.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
[37/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\bank_conflict.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[38/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\ir\tensor.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[39/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\alloc.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[40/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\dpas_atomic.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[41/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\dp4a.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[42/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\barrier.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[43/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\hoist.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[44/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\cse.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[45/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\overflow.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[46/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\dpasw.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[47/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\simplify.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[48/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\slm.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[49/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\shuffle_splitter.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[50/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\strength_reduce.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[51/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\peephole.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[52/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\pass.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[53/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\send.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[54/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\pass\unroll.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[55/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\reorder\ir_builder.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[56/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\utils\trace.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[57/380] Building CXX object src\gpu\jit\CMakeFiles\dnnl_gpu_jit.dir\reorder\gen_reorder.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[58/380] Generating gen9_bnorm_header.cpp

[59/380] Generating binary_types_header.cpp

[60/380] Generating ocl_gemm_attrs_header.cpp

[61/380] Generating concat_common_header.cpp

[62/380] Generating metrics_discovery_api_header.cpp

[63/380] Generating ocl_math_utils_header.cpp

[64/380] Generating ocl_scales_header.cpp

[65/380] Generating offsets_header.cpp

[66/380] Generating ocl_zero_points_header.cpp

[67/380] Generating gemm_with_post_ops_kernel.cpp

[68/380] Generating ocl_eltwise_header.cpp

[69/380] Generating rnn_types_header.cpp

[70/380] Generating combined_reduction_kernel.cpp

[71/380] Generating ocl_types_header.cpp

[72/380] Generating zero_pad_struct_header.cpp

[73/380] Generating ocl_post_ops_header.cpp

[74/380] Generating reorder_common_header.cpp

[75/380] Generating custom_reorder_kernel.cpp

[76/380] Generating ref_gemm_kernel.cpp

[77/380] Generating gemm_post_ops_inner_product_kernel.cpp

[78/380] Generating gen9_concat_kernel.cpp

[79/380] Generating gen9_binary_kernel.cpp

[80/380] Generating xe_hp_systolic_gemm_copy_kernel.cpp

[81/380] Generating gen9_conv_bwd_weights_kernel.cpp

[82/380] Generating gen9_bnorm_nhwc_bwd_kernel.cpp

[83/380] Generating gen9_bnorm_bwd_kernel.cpp

[84/380] Generating gen9_conv_dw_fwd_data_kernel.cpp

[85/380] Generating gen9_bnorm_reduce_kernel.cpp

[86/380] Generating gen9_conv_dw_bwd_data_kernel.cpp

[87/380] Generating gen9_conv_bwd_data_kernel.cpp

[88/380] Generating gen9_bnorm_nhwc_fwd_kernel.cpp

[89/380] Generating xe_hpc_systolic_gemm_copy_kernel.cpp

[90/380] Generating gen9_bnorm_fwd_kernel.cpp

[91/380] Generating gen9_conv_nhwc_bwd_data_kernel.cpp

[92/380] Generating gen9_eltwise_kernel.cpp

[93/380] Generating gen9_wino_conv_fwd_data_fused_kernel.cpp

[94/380] Generating gen9_conv_nhwc_fwd_data_kernel.cpp

[95/380] Generating gen9_sum_kernel.cpp

[96/380] Generating gen9_wino_conv_fwd_data_2x3_kernel.cpp

[97/380] Generating gen9_conv_nhwc_bwd_weights_f32_kernel.cpp

[98/380] Generating generic_reorder_kernel.cpp

[99/380] Generating gen9_pooling_kernel.cpp

[100/380] Generating gen9_softmax_kernel.cpp

[101/380] Generating ref_inner_product_kernel.cpp

[102/380] Generating gen9_global_pooling_kernel.cpp

[103/380] Generating gen9_conv_fwd_data_kernel.cpp

[104/380] Generating ref_convolution_kernel.cpp

[105/380] Generating ref_matmul_kernel.cpp

[106/380] Generating ref_lrn_kernel.cpp

[107/380] Generating ref_prelu_kernel.cpp

[108/380] Generating ref_binary_kernel.cpp

[109/380] Generating ref_pooling_kernel.cpp

[110/380] Generating ref_deconv_backward_bias_kernel.cpp

[111/380] Generating ref_eltwise_kernel.cpp

[112/380] Generating ref_reduction_kernel.cpp

[113/380] Generating ref_bnorm_kernel.cpp

[114/380] Generating many_inputs_sum_kernel.cpp

[115/380] Generating ref_layer_normalization_kernel.cpp

[116/380] Generating ref_shuffle_kernel.cpp

[117/380] Generating simple_sum_kernel.cpp

[118/380] Generating rnn_reorder_kernel.cpp

[119/380] Generating ref_rnn_kernel.cpp

[120/380] Generating ref_resampling_kernel.cpp

[121/380] Generating ref_reorder_kernel.cpp

[122/380] Generating simple_concat_kernel.cpp

[123/380] Generating ref_softmax_kernel.cpp

[124/380] Generating vectorized_lnorm_kernel.cpp

[125/380] Generating xe_lp_conv_bwd_data_mb_block_x8s8x8_kernel.cpp

[126/380] Generating vectorized_resampling_kernel.cpp

[127/380] Generating xe_lp_1x1_conv_fwd_data_x8s8x_kernel.cpp

[128/380] Generating ref_zero_pad_kernel.cpp

[129/380] Generating xe_lp_conv_fwd_data_first_x8s8x_kernel.cpp

[130/380] Generating xe_lp_conv_fwd_data_mb_block_x8s8x_kernel.cpp

[131/380] Generating xe_lp_conv_nhwc_fwd_dw_mb_block_x8s8x_kernel.cpp

[132/380] Generating xe_lp_conv_bwd_data_x8s8x8_kernel.cpp

[133/380] Generating xe_lp_nhwc_1x1_conv_fwd_x8s8x_kernel.cpp

[134/380] Generating xe_lp_x8s8x_compensation_kernel.cpp

[135/380] Generating xe_lp_conv_dw_fwd_data_ow_block_x8s8x_kernel.cpp

[136/380] Generating xe_lp_conv_nhwc_fwd_dw_ow_block_x8s8x_kernel.cpp

[137/380] Generating xe_lp_conv_fwd_data_ow_block_x8s8x_kernel.cpp

[138/380] Generating xe_lp_conv_nhwc_fwd_x8s8x_kernel.cpp

[139/380] Generating xe_lp_conv_dw_fwd_data_mb_block_x8s8x_kernel.cpp

[140/380] Generating xe_lp_conv_nhwc_fwd_first_x8s8x_kernel.cpp

[141/380] Building CXX object src\common\CMakeFiles\dnnl_common.dir\convolution.cpp.obj

FAILED: src/common/CMakeFiles/dnnl_common.dir/convolution.cpp.obj 

C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe  /nologo /TP -DCL_TARGET_OPENCL_VERSION=120 -DDNNL_DISABLE_PRIMITIVE_CACHE -DDNNL_ENABLE_CONCURRENT_EXEC -DDNNL_ENABLE_CPU_ISA_HINTS -DDNNL_ENABLE_MAX_CPU_ISA -DDNNL_X64=1 -DNOMINMAX -DWIN32_LEAN_AND_MEAN -D_WIN -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\include -IC:\CYH\vcpkg\buildtrees\openvino\x64-windows-dbg\src\plugins\intel_gpu\thirdparty\onednn_gpu_build\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src -IC:\CYH\vcpkg\installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /wd4996 /wd4244 /wd4018 -openmp  /MP /bigobj /Zc:preprocessor    /wd4800 /wd4068 /wd4305 /wd4551 /wd4244 /wd5105  -DITT_API_IPT_SUPPORT /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /Fosrc\common\CMakeFiles\dnnl_common.dir\convolution.cpp.obj /Fdsrc\common\CMakeFiles\dnnl_common.dir\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\convolution.cpp

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\convolution.cpp(46): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\convolution.cpp(49): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\convolution.cpp(76): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\convolution.cpp(135): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\convolution.cpp(137): error C2059: syntax error: ','
[142/380] Building CXX object src\common\CMakeFiles\dnnl_common.dir\concat.cpp.obj

FAILED: src/common/CMakeFiles/dnnl_common.dir/concat.cpp.obj 

C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe  /nologo /TP -DCL_TARGET_OPENCL_VERSION=120 -DDNNL_DISABLE_PRIMITIVE_CACHE -DDNNL_ENABLE_CONCURRENT_EXEC -DDNNL_ENABLE_CPU_ISA_HINTS -DDNNL_ENABLE_MAX_CPU_ISA -DDNNL_X64=1 -DNOMINMAX -DWIN32_LEAN_AND_MEAN -D_WIN -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\include -IC:\CYH\vcpkg\buildtrees\openvino\x64-windows-dbg\src\plugins\intel_gpu\thirdparty\onednn_gpu_build\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src -IC:\CYH\vcpkg\installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /wd4996 /wd4244 /wd4018 -openmp  /MP /bigobj /Zc:preprocessor    /wd4800 /wd4068 /wd4305 /wd4551 /wd4244 /wd5105  -DITT_API_IPT_SUPPORT /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /Fosrc\common\CMakeFiles\dnnl_common.dir\concat.cpp.obj /Fdsrc\common\CMakeFiles\dnnl_common.dir\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\concat.cpp

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\concat.cpp(45): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\concat.cpp(54): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\concat.cpp(69): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\concat.cpp(88): error C2059: syntax error: ','
[143/380] Building CXX object src\common\CMakeFiles\dnnl_common.dir\cache_blob_id.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[144/380] Building CXX object src\common\CMakeFiles\dnnl_common.dir\batch_normalization.cpp.obj

FAILED: src/common/CMakeFiles/dnnl_common.dir/batch_normalization.cpp.obj 

C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe  /nologo /TP -DCL_TARGET_OPENCL_VERSION=120 -DDNNL_DISABLE_PRIMITIVE_CACHE -DDNNL_ENABLE_CONCURRENT_EXEC -DDNNL_ENABLE_CPU_ISA_HINTS -DDNNL_ENABLE_MAX_CPU_ISA -DDNNL_X64=1 -DNOMINMAX -DWIN32_LEAN_AND_MEAN -D_WIN -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\include -IC:\CYH\vcpkg\buildtrees\openvino\x64-windows-dbg\src\plugins\intel_gpu\thirdparty\onednn_gpu_build\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src -IC:\CYH\vcpkg\installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /wd4996 /wd4244 /wd4018 -openmp  /MP /bigobj /Zc:preprocessor    /wd4800 /wd4068 /wd4305 /wd4551 /wd4244 /wd5105  -DITT_API_IPT_SUPPORT /MDd /Zi /Ob0 /Od /RTC1 /showIncludes /Fosrc\common\CMakeFiles\dnnl_common.dir\batch_normalization.cpp.obj /Fdsrc\common\CMakeFiles\dnnl_common.dir\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\batch_normalization.cpp

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\batch_normalization.cpp(43): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\batch_normalization.cpp(46): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\batch_normalization.cpp(47): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\batch_normalization.cpp(49): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\batch_normalization.cpp(58): error C2059: syntax error: ','
C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\batch_normalization.cpp(77): error C2059: syntax error: ','
[145/380] Building CXX object src\common\CMakeFiles\dnnl_common.dir\convolution_pd.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
[146/380] Building CXX object src\gpu\ocl\CMakeFiles\dnnl_gpu_ocl.dir\capi\engine.cpp.obj

cl : Command line warning D9002 : ignoring unknown option '/Zc:preprocessor'
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(364): warning C4307: '+': integral constant overflow
c:\cyh\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\onednn_gpu\src\common\memory_tracking.hpp(444): warning C4307: '+': integral constant overflow
ninja: build stopped: subcommand failed.

[1916/3114] C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe   /TP -DCL_TARGET_OPENCL_VERSION=200 -DIE_BUILD_POSTFIX=\"d\" -DIE_THREAD=IE_THREAD_TBB -DIN_OV_COMPONENT -DOV_GPU_OPENCL_HPP_HAS_UUID -DOV_GPU_USE_OPENCL_HPP -DOV_NATIVE_PARENT_PROJECT_ROOT_DIR=\"2023.1.0-d4639b6ba4.clean\" -DOV_THREAD=OV_THREAD_TBB -DTBB_PREVIEW_WAITING_FOR_WORKERS=1 -DTBB_USE_DEBUG -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector\kernels -IC:\CYH\vcpkg\buildtrees\openvino\x64-windows-dbg\src\plugins\intel_gpu\src\kernel_selector\codegen\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\rapidjson -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\preprocessing\src -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\core\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\frontends\common\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\include\ie -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\core\dev_api -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\transformations\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\itt\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\util\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\dev_api -IC:\CYH\vcpkg\installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /wd4996 /wd4244 /wd4018 /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -std:c++14 -MDd /d1trimfile:C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\ /d1trimfile:C:/CYH/vcpkg/buildtrees/openvino/src/2023.1.0-d4639b6ba4.clean/ /showIncludes /Fosrc\plugins\intel_gpu\src\kernel_selector\CMakeFiles\openvino_intel_gpu_kernels.dir\kernels\reorder\reorder_kernel_fast_b1.cpp.obj /FdC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\bin\intel64\Debug\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector\kernels\reorder\reorder_kernel_fast_b1.cpp

[1917/3114] C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe   /TP -DCL_TARGET_OPENCL_VERSION=200 -DIE_BUILD_POSTFIX=\"d\" -DIE_THREAD=IE_THREAD_TBB -DIN_OV_COMPONENT -DOV_GPU_OPENCL_HPP_HAS_UUID -DOV_GPU_USE_OPENCL_HPP -DOV_NATIVE_PARENT_PROJECT_ROOT_DIR=\"2023.1.0-d4639b6ba4.clean\" -DOV_THREAD=OV_THREAD_TBB -DTBB_PREVIEW_WAITING_FOR_WORKERS=1 -DTBB_USE_DEBUG -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector\kernels -IC:\CYH\vcpkg\buildtrees\openvino\x64-windows-dbg\src\plugins\intel_gpu\src\kernel_selector\codegen\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\rapidjson -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\preprocessing\src -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\core\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\frontends\common\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\include\ie -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\core\dev_api -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\transformations\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\itt\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\util\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\dev_api -IC:\CYH\vcpkg\installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /wd4996 /wd4244 /wd4018 /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -std:c++14 -MDd /d1trimfile:C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\ /d1trimfile:C:/CYH/vcpkg/buildtrees/openvino/src/2023.1.0-d4639b6ba4.clean/ /showIncludes /Fosrc\plugins\intel_gpu\src\kernel_selector\CMakeFiles\openvino_intel_gpu_kernels.dir\kernels\reorder\reorder_kernel_fs_b_yx_fsv32_to_bfyx.cpp.obj /FdC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\bin\intel64\Debug\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector\kernels\reorder\reorder_kernel_fs_b_yx_fsv32_to_bfyx.cpp

[1918/3114] C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\cl.exe   /TP -DCL_TARGET_OPENCL_VERSION=200 -DIE_BUILD_POSTFIX=\"d\" -DIE_THREAD=IE_THREAD_TBB -DIN_OV_COMPONENT -DOV_GPU_OPENCL_HPP_HAS_UUID -DOV_GPU_USE_OPENCL_HPP -DOV_NATIVE_PARENT_PROJECT_ROOT_DIR=\"2023.1.0-d4639b6ba4.clean\" -DOV_THREAD=OV_THREAD_TBB -DTBB_PREVIEW_WAITING_FOR_WORKERS=1 -DTBB_USE_DEBUG -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector\kernels -IC:\CYH\vcpkg\buildtrees\openvino\x64-windows-dbg\src\plugins\intel_gpu\src\kernel_selector\codegen\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\thirdparty\rapidjson -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\preprocessing\src -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\core\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\frontends\common\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\include\ie -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\core\dev_api -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\transformations\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\itt\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\common\util\include -IC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\inference\dev_api -IC:\CYH\vcpkg\installed\x64-windows\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_CRT_SECURE_NO_WARNINGS /D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /W3 /bigobj /MP /wd4251 /wd4275 /wd4996 /wd4244 /wd4018 /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -std:c++14 -MDd /d1trimfile:C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\ /d1trimfile:C:/CYH/vcpkg/buildtrees/openvino/src/2023.1.0-d4639b6ba4.clean/ /showIncludes /Fosrc\plugins\intel_gpu\src\kernel_selector\CMakeFiles\openvino_intel_gpu_kernels.dir\kernels\reorder\reorder_kernel_to_yxfb_batched.cpp.obj /FdC:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\bin\intel64\Debug\ /FS -c C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\src\plugins\intel_gpu\src\kernel_selector\kernels\reorder\reorder_kernel_to_yxfb_batched.cpp

[1919/3114] cmd.exe /C "cd . && C:\PROGRA~2\MICROS~4\2017\COMMUN~1\VC\Tools\MSVC\1416~1.270\bin\Hostx64\x64\lib.exe  /machine:x64 /nologo /out:C:\CYH\vcpkg\buildtrees\openvino\src\2023.1.0-d4639b6ba4.clean\bin\intel64\Debug\openvino_onednn_cpud.lib @CMakeFiles\dnnl.rsp  && cd ."

ninja: build stopped: subcommand failed.

```
</details>

### Additional context

_No response_

--- a/ports/opencv4/portfile.cmake
+++ b/ports/opencv4/portfile.cmake
@@ -71,6 +71,7 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
  "jpeg"      WITH_JPEG
  "lapack"    WITH_LAPACK
  "nonfree"   OPENCV_ENABLE_NONFREE
+ "openvino"  WITH_OPENVINO
  "openexr"   WITH_OPENEXR
  "opengl"    WITH_OPENGL
  "ovis"      CMAKE_REQUIRE_FIND_PACKAGE_OGRE
@@ -86,13 +87,9 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
 
 # Cannot use vcpkg_check_features() for "dnn", "gtk", ipp", "openmp", "ovis", "python", "qt", "tbb"
 set(BUILD_opencv_dnn OFF)
-set(WITH_OPENVINO OFF)
 if("dnn" IN_LIST FEATURES)
   if(NOT VCPKG_TARGET_IS_ANDROID)
     set(BUILD_opencv_dnn ON)
-    if(NOT VCPKG_TARGET_IS_UWP)
-      set(WITH_OPENVINO ON)
-    endif()
   else()
     message(WARNING "The dnn module cannot be enabled on Android")
   endif()

--- a/ports/opencv4/vcpkg.json
+++ b/ports/opencv4/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "opencv4",
   "version": "4.8.0",
-  "port-version": 8,
+  "port-version": 9,
   "description": "computer vision library",
   "homepage": "https://github.com/opencv/opencv",
   "license": "Apache-2.0",
@@ -108,10 +108,6 @@
           "name": "flatbuffers",
           "host": true,
           "default-features": false
-        },
-        {
-          "name": "openvino",
-          "platform": "!uwp"
         },
         "protobuf"
       ]
@@ -246,6 +242,37 @@
     },
     "openmp": {
       "description": "Enable openmp support for opencv"
+    },
+    "openvino": {
+      "description": "OpenVINO support for OpenCV DNN",
+      "supports": "!uwp",
+      "dependencies": [
+        {
+          "name": "opencv4",
+          "default-features": false,
+          "features": [
+            "dnn"
+          ]
+        },
+        {
+          "name": "openvino",
+          "default-features": false,
+          "features": [
+            "auto",
+            "cpu",
+            "hetero"
+          ],
+          "platform": "!uwp"
+        },
+        {
+          "name": "openvino",
+          "default-features": false,
+          "features": [
+            "gpu"
+          ],
+          "platform": "x64 & !(osx | uwp) & !static"
+        }
+      ]
     },
     "ovis": {
       "description": "opencv_ovis module",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6198,7 +6198,7 @@
     },
     "opencv4": {
       "baseline": "4.8.0",
-      "port-version": 8
+      "port-version": 9
     },
     "opendnp3": {
       "baseline": "3.1.1",

--- a/versions/o-/opencv4.json
+++ b/versions/o-/opencv4.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "7ed8c48a9b2be5df262ccbcfa876f5314f429c10",
+      "version": "4.8.0",
+      "port-version": 9
+    },
+    {
       "git-tree": "48c97b54fceaef9a96d374693f47e3ea91383f3c",
       "version": "4.8.0",
       "port-version": 8

